core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -56,11 +56,12 @@ import scala.collection._
  *                          --entity-type users --entity-name <user-principal> --entity-type clients --entity-name <client-id>
  *     <li> broker: --broker <broker-id> OR --entity-type brokers --entity-name <broker-id>
  *     <li> broker-logger: --broker-logger <broker-id> OR --entity-type broker-loggers --entity-name <broker-id>
+ *     <li> ip: --IP <ip> OR --entity-type IPs --entity-name <ip>
  * </ul>
- * --entity-type <users|clients|brokers> --entity-default may be specified in place of --entity-type <users|clients|brokers> --entity-name <entityName>
- * when describing or altering default configuration for users, clients, or brokers, respectively.
- * Alternatively, --user-defaults, --client-defaults, or --broker-defaults may be specified in place of
- * --entity-type <users|clients|brokers> --entity-default, respectively.
+ * --entity-type <users|clients|brokers|IPs> --entity-default may be specified in place of --entity-type <users|clients|brokers|IPs> --entity-name <entityName>
+ * when describing or altering default configuration for users, clients, brokers, or IPs, respectively.
+ * Alternatively, --user-defaults, --client-defaults, --broker-defaults, or --IP-defaults may be specified in place of
```

Comment:
`--IP-defaults` looks pretty bad. I think that we should definitely use lower-case.

Reply:
sounds good. In this case, do we want to change all mentions of IP to lower-case? including entity type/ZK path.

Reply:
yes, agreed about `--IP-defaults` looking bad. 

The main reason I originally proposed upper case "IP" in the KIP is because `ips` as in `--entity-type ips` does not seem very recognizable as plural for IP (but that could be just me), i.e., not very user friendly. But if we do `--entity-type IPs` and then `--ip-defaults`, I think that would cause more issues later because people would assume it's consistent and use wrong case.

Ok, I am convinced that we lower-case is better, since there are more advantages to this vs. upper-case.

Reply:
@splett2 Isn't the ZK path already lower-case?

Reply:
@dajac 
it was originally lower-case in #9386
after reading the KIP, i realized it was described as being upper case, e.g.
```
Default quota for <IPs> will be stored in Zookeeper at /config/IPs/<default>
Quota for a specific IP address for which quota override is defined will be stored in Zookeeper at /config/IPs/<ip-address>
```
and changed it to upper in this PR with the change in
```
core/src/main/scala/kafka/server/DynamicConfigManager.scala
```